src/test/java/no/unit/nva/bare/UpdateAuthorityIdentifierHandlerTest.java

```diff
@@ -221,6 +230,39 @@ public void handlerReturnsOkWhenInputIsValidAndAuthorityIdentifierIsUpdatedSucce
         assertEquals(SC_OK, gatewayResponse.getStatusCode());
     }
 
+    @Test
+    @DisplayName("handler Returns Ok Response When Input Is Valid URI And Authority Identifier Is Updated Successfully")
+    public void handlerReturnsOkWhenInputIsValidUriAndAuthorityIdentifierIsUpdatedSuccessfully() throws Exception {
+
+        InputStream is =
+                UpdateAuthorityIdentifierHandler.class.getResourceAsStream(BARE_SINGLE_AUTHORITY_GET_RESPONSE_JSON);
+        final BareAuthority bareAuthority = new Gson().fromJson(new InputStreamReader(is), BareAuthority.class);
+        Config.getInstance().setBareHost("localhost");
+        HttpClient httpClient = mock(HttpClient.class);
```

Comment:
can you extract the mock creation to a suitably named method?

Reply:
Extracted two methods, one for bareConnection and one for updateHandler